docs/sources/reference/components/prometheus/prometheus.write.queue.md

```diff
@@ -117,6 +116,31 @@ Name                   | Type     | Description
 `insecure_skip_verify` | `bool`   | Disables validation of the server certificate.           |         | no
 `key_pem`              | `secret` | Key PEM-encoded text for client authentication.          |         | no
 
+
+### parallelism block
+
+Name            | Type       | Description                                                                                                                        | Default | Required
+----------------|------------|------------------------------------------------------------------------------------------------------------------------------------|---------|---------
+`drift_scale_up_seconds`      | `int`      | The maximum amount of time between the timestamps of incoming signals and outgoing signals before increasing desired connections.  | `60`    | no
+`drift_scale_down_seconds`    | `int`      | The minimum amount of time between the timestamps of incoming signals and outgoing signals before descreasing desired connections. | `30`    | no
```

Comment:
I'd prefer this to be duration to stay consistent with Alloy's configs everywhere else. We also have duration used in this config block already for flush interval.